inflect.py

```diff
@@ -1381,15 +1390,26 @@ def _get_value_from_ast(self, obj):
         '''
         if isinstance(obj, ast.Num):
             return obj.n
-        elif isinstance(obj, ast.Name):
-            return obj.id
         elif isinstance(obj, ast.Str):
             return obj.s
         elif isinstance(obj, ast.List):
             return [self._get_value_from_ast(e) for e in obj.elts]
         elif isinstance(obj, ast.Tuple):
             return tuple([self._get_value_from_ast(e) for e in obj.elts])
 
+        # None, True and False are NameConstants in Py3.4 and above.
+        elif sys.version_info >= (3, 4) and isinstance(obj, ast.NameConstant):
```

Comment:
inflect.py supports Python 2.7 and 3.4, so this could be written as:

```python
sys.version_info.major >= 3
```

Reply:
Okay, will make the change